cypress/integration/main.spec.js

```diff
@@ -0,0 +1,59 @@
+import { setData } from '../../src/js/Manager/SaveVideoManager';
+import { FAKE_DATA, SAVE_KEY } from '../../src/js/constants';
+
+describe('저장된 비디오를 메인 화면에서 확인할 수 있다.', () => {
+  it('저장된 동영상이 있을 경우, 볼 영상에서 확인할 수 있다.', () => {
+    const videoId = '3iM_06QeZi8';
+    setData(SAVE_KEY, FAKE_DATA);
+    cy.visit('/index.html');
+    cy.get(`.video-item[data-video-id=${videoId}]`).should('exist');
+  });
+
+  it('저장된 동영상이 없는 경우 이를 사용자에게 알려준다.', () => {
+    cy.visit('/index.html');
+    cy.get(`.no-saved-video__image`).should('be.visible');
+  });
+});
+
+describe('볼 동영상에서 본 동영상으로 변환할 수 있다.', () => {
+  const videoId = '3iM_06QeZi8';
+  it('체크 이모지 버튼을 누르면 본 동영상으로 전환된다.', () => {
+    setData(SAVE_KEY, FAKE_DATA);
+    cy.visit('/index.html');
+    cy.get(`.video-item[data-video-id=${videoId}] > div > button.video-item__watched-button.button`).click();
```

Comment:
```suggestion
    cy.get(`.video-item[data-video-id=${videoId}] .video-item__watched-button`).click();
```
selector 를 dom 구조에 따라 작성할 필요가 없지 않을까요?